text/0225-configuration.md

```diff
@@ -0,0 +1,375 @@
+
+### What is the exact configuration file format to use?
+
+Included in this OTEP is an example configuration file format.
+This included format was settled on so the configuration file schemas being proposed here could all be evaluated as viable options.
+It acted as proof-of-concept that _a_ configuration file format existed that could describe needed OTel configuration and be described by a schema.
+However, the configuration file format presented here is not meant as the final, nor optimal, design for use by OpenTelemetry.
+
+What that final design will be is left to discussion when this OTEP is implemented in the OpenTelemetry specification.
+It is explicitly something this OTEP is not intended to resolved.
+
+The following are existing questions that will need to be resolved in the final design:
+
+1. Should the trace exporter be at the same level as the span processors?
+2. Should the sampler config be separate from the sampler?
+3. Is the `sdk` key appropriate? Should alternate configuration live under its own key but the SDKs configuration be at the top level?
+4. Should the `disabled` key be renamed as `enabled`?
+
+This list is not intended as comprehensive.
+There are likely more questions related to the final design that will be discussed when implemented in the OpenTelemetry specification.
+
+## Future possibilities
+
+### Additional configuration providers
+
+Although the initial proposal for configuration supports only describes in-code and file representations, it's possible additional sources (remote, opamp, ...) for configuration will be desirable. The implementation of the configuration model and components should be extensible to allow for this.
+
+### Integration with auto-instrumentation
+
+The configuration model could be integrated to work with the existing auto-instrumentation tooling in each language implementation.
+
+#### Java
+
+The Java implementation provides a JAR that supports configuring various parameters via system properties. This implementation could leverage a configuration file by supporting its configuration a system property:
+
+```bash
+java -javaagent:path/to/opentelemetry-javaagent.jar \
+     -Dotel.config=./config.yaml
+     -jar myapp.jar
+```
+
+#### Python
+
+The Python implementation has a command available that allows users to leverage auto-instrumentation. The `opentelemetry-instrument` command could use a `--config` flag to pass in a config file:
+
+```bash
+# install the instrumentation package
+pip install opentelemetry-instrumentation
+# use a --config parameter to pass in the configuration file
+# NOTE: this parameter does not currently exist and would need to be added
+opentelemetry-instrument --config ./config.yaml ./python/app.py
```

Comment:
We give guidance for a default name and location or do we plan to always require explicit location?  I support the file path being explicit, but perhaps that should be called out as a requirement?

Reply:
I think this makes sense to call out in the specification PRs 👍🏻